server/debug.go

```diff
@@ -29,6 +29,39 @@ func (s *Server) registerDebugRoutes(g *echo.Group) {
 
 		return currentDebugState(c)
 	})
+
+	g.GET("/debug/log", func(c echo.Context) error {
+		var errorRecordList []*api.DebugLog
+		var incrID int
+
+		s.errorRecordRing.Mutex.RLock()
+		defer s.errorRecordRing.Mutex.RUnlock()
+
+		s.errorRecordRing.Ring.Do(func(p interface{}) {
+			if p != nil {
+				incrID++
+				log := &api.DebugLog{
+					ID: incrID,
```

Comment:
1) Should incrID start with zero instead of one?

2) nit: use the variable name id instead, or incrementID.